WordPress/gutenberg
# [RNMobile] Show "Add to beginning" and "Add to end" option always

## Description

Closes https://github.com/wordpress-mobile/gutenberg-mobile/issues/2344 by making the long-press options **Add to beginning** and **Add to end** available in all scenarios. Prior to this PR these options were only available when no blocks were selected. 

### Related PR
`gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3074

## How has this been tested?



1. Open the demo app
2. Long-press on the inserter button, verify options **Add to beginning** and **Add to end** are displayed. 
3. Tap on **Add to beginning** and select a block. Verify block added to the beginning. 
4. With no block selected, long-press on inserter again, this time tap on **Add to end** and select a block. Verify added to end of post. 
5. Select a block in the editor, then long-press on the inserter. Verify new options **Add to beginning** and **Add to end** are displayed.
6. Tap on **Add to beginning** and select a block. Verify added to beginning of document. 
7. Repeat for the other options available when long-pressing on inserter with a block selected and verify each work. 

## Screenshots 
No block selected BEFORE | AFTER
-- | --
![long-press-before](https://user-images.githubusercontent.com/5810477/106337257-bb138a80-625e-11eb-8ce9-3fb95eee2f78.png)|![long-press-after](https://user-images.githubusercontent.com/5810477/106337258-bcdd4e00-625e-11eb-9cf1-f43651729667.png)

Block selected BEFORE | AFTER
-- | --
![long-press-focused-before](https://user-images.githubusercontent.com/5810477/106337281-cb2b6a00-625e-11eb-984d-35d1921caf34.png)|![long-press-focused-after](https://user-images.githubusercontent.com/5810477/106337289-ce265a80-625e-11eb-92b5-d51df3dcebf1.png)



## Types of changes




- Updated logic to show the **Add to beginning** and **Add to end option** always.
- Changed the icon for **Add block before** and **Add block after** to `plusCircle` per [design recommendation here](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2344#issuecomment-769289639).

## Checklist:
